### PR TITLE
Implement drawtable function

### LINKED ISSUE #193: drawtable
We should implement [the `drawtable` operator](http://cindyjs.org/ref/Texts_and_Tables.html#drawtable$2) in a way compatible to Cinderella.

### PR DESCRIPTION
**Pull request by gagern**
Defaults and size factors are taken from Cinderella.

This fixes #193 (although not in the way discussed there).
Don't forget to tick the function in #219 as well.

### CODE REVIEW

File: ref/Texts_and_Tables.md [hunk 1]
```diff
@@ -150,18 +150,15 @@ It returns a boolean value.
...
 Modifiers can be used to fine tune the appearance of the table.
 
 **Example:**
 The following code produces the picture below:
 
-    - skip test: drawtable not implemented yet
-    > x=1..10;
-    > table=apply(x,(#,#^2,#^3,#^4));
+    > x=1..4;
+    > table=apply(x,(#,#^2,#^3));
```
Comment (montaga) [inline ref/Texts_and_Tables.md]: This code does not produce build/ref/img/Table1X.png

**Comment by montaga** [comment]
The code and the examples look fine to me. Nethertheless, the image in the references should match the code in the reference.

File: ref/Texts_and_Tables.md (hunk 1 above)
Comment (gagern) [inline ref/Texts_and_Tables.md]: I guess that I had thought about adding draw command tests here, and wanted to keep the number of commands low, but then decided against it again. By the way, this just made me think of #460, an idea which I've been toying with for quite some time now.

**Comment by gagern** [comment]
I restored the previous code to match the figure.

**Pull request merged**